### DISCUSSION TITLE
Help Wanted!!!

### DISCUSSION THREAD

**Discussion by jsixface** [quillpad/quillpad#280] [General]
I had to switch to an iPhone couple of month and won't be able to test the Quillpad app on my phone anymore. I am still an Android developer by job, though not mostly on the UI. I am looking for some help in maintaining this repo. I'd like volunteers to take partial ownership of this github repo. The responsibilities will be to test the newer releases on the phones since we don't have a test driven build. I would also appreciate some help in reviewing the pull requests, grooming the issues and taking the project forward. 

I can be reached out in the Matrix chat room https://matrix.to/#/%23quillpad_community:gitter.im

**Comment by clach04** [comment]
@jsixface is an iOS build in the works? https://github.com/quillpad/quillpad/discussions/55 indicated maybe. I'm an android user (but not a developer) so I'm not looking for an iOS build :-)

I can start watching releases as a possible tester. As it stands quillpad is missing features I want/need to use as my daily driver but I'm not a position to contribute code to remedy that.

**Comment by jsixface** [comment]
iOS app was initially on the roadmap but after taking a deeper look at the code, i think it won't be an easy task. Most of the libraries used here are not multi platform compatible.

**Comment by RustoMCSpit** [comment]
it woulf be a good idea to post this on reddit and lemmy

**Comment by RustoMCSpit** [comment]
> iOS app was initially on the roadmap but after taking a deeper look at the code

has the roadmap been changed?

**Comment by erAck** [comment]
> it woulf be a good idea to post this on reddit and lemmy

This is how one attracts and invites Jia Tan and Hans Jansen types.

**Comment by squatica** [comment]
To become the new maintainer, please click the "Fork" button.

Seriously guys, have we not learned anything? By adding maintainers to an existing project with users and distribution channels, you are handing out those channels too. Why risk it? This is what forks are for.

**Comment by RustoMCSpit** [comment]
> > it woulf be a good idea to post this on reddit and lemmy
> 
> This is how one attracts and invites Jia Tan and Hans Jansen types.

this isnt a huge dependency of a large piece of software

**Comment by erAck** [comment]
> > > it woulf be a good idea to post this on reddit and lemmy
> > 
> > 
> > This is how one attracts and invites Jia Tan and Hans Jansen types.
> 
> this isnt a huge dependency of a large piece of software

Malicious account holders with a long term goal need to build reputation. It doesn't matter much that such an app isn't a dependency of other software.

**Comment by squatica** [comment]
> this isnt a huge dependency of a large piece of software

Ah ok, in that case it's no problem if it gets backdoored... is that what you mean?

The xz story isn't even the first one, remember event-stream? https://old.reddit.com/r/javascript/comments/a158zr/dominics_response_to_the_eventstream_npm_package/ - please, if you can't continue maintaining your project, just set it as Archived, don't gamble with your users without their consent.

**Comment by jsixface** [comment]
Oh boy. Just now caught up with the news. What a week. 

I wanted to clarify that I wasn't planning on transferring the ownership. Just  a little more help so that PRs like #240 won't be sitting there waiting for someone to test it on a real device. 

Edit: I'm gonna transfer this to a discussion instead of an issue

**Reply by nofishonfriday** [reply]
> Just a little more help so that PRs like https://github.com/quillpad/quillpad/pull/240 won't be sitting there waiting for someone to test it on a real device.

For this it would be nice if the build artefacts could be auto-uploaded, as suggested in that PR.

**Reply by jsixface** [reply]
Github Actions now produce APKs for PRs that can be downloaded and tested.

**Comment by RustoMCSpit** [comment]
can we re-pin the markdown issue as the issue for this was closed

**Reply by jsixface** [reply]
Can you link the issue? I don't get which one you're referring to.
